server/services/interventionsService.test.ts

```diff
@@ -1069,6 +1072,43 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
       expect(await interventionsService.getSentReferrals(token)).toEqual([sentReferral, sentReferral])
     })
   })
+
+  describe('getInterventions', () => {
+    it('returns a list of all interventions', async () => {
+      const intervention = {
+        id: '15237ae5-a017-4de6-a033-abf350f14d99',
+        title: 'Better solutions (anger management)',
+        description:
+          'To provide service users with key tools and strategies to address issues of anger management and temper control and explore the link between thoughts, emotions and behaviour. It provides the opportunity for service users to practice these strategies in a safe and closed environment.',
+        pccRegions: [
+          { id: 'cheshire', name: 'Cheshire' },
+          { id: 'cumbria', name: 'Cumbria' },
+          { id: 'lancashire', name: 'Lancashire' },
+          { id: 'merseyside', name: 'Merseyside' },
+        ],
+        serviceCategory: serviceCategoryFactory.build(),
```

Comment:
🙋‍♂️ I've noticed [on the broker page](https://pact-broker-prod.apps.live-1.cloud-platform.service.justice.gov.uk/pacts/provider/Interventions%20Service/consumer/Interventions%20UI/version/03f3a49c505a0adcbd46f0639cb5d78ac4fde626#a_request_for_all_interventions_given_There_are_some_interventions) that this includes `desiredOutcomes` and `complexityLevels`.

Currently, this makes it harder to see what is the minimum data the UI _needs_ from the backend. What do you think about only specifying the structure that is required for the search results to work?